sam-karis/airtech
# #add user profile

## What does this PR do?
------
It adds user profile model, user can edit, upload and delete their profile image

## Description of Task to be completed?
-----
- add profile model, serializers, URLs, and views
- update test
- add base model

## How should this be manually tested?
------
clone the repo
check out to `ft-upload-image-167660081`
add environment variables as defined in [`.env_example`](https://github.com/sam-karis/airtech/blob/develop/.env_example)
Create a database and run `python manage.py migrate` to create relevant tables.


## Any background context you want to provide?
------
None

## What are the relevant pivotal tracker stories?
------
[login user](https://www.pivotaltracker.com/story/show/167660081)


## Screenshots (if appropriate)
------
#### View users profile
![image](https://user-images.githubusercontent.com/22454909/63216143-ce4d9b80-c139-11e9-9cc0-996b9974b19b.png)



#### Edit user profile/image
![image](https://user-images.githubusercontent.com/22454909/63216123-71ea7c00-c139-11e9-8353-3cc2d8abd2a8.png)


#### Delete profile image
![image](https://user-images.githubusercontent.com/22454909/63216099-fe486f00-c138-11e9-8ac8-4794d4ccdf91.png)

![image](https://user-images.githubusercontent.com/22454909/63216129-8af32d00-c139-11e9-9cfd-f2a36d060b05.png)
